CHANGELOG.rst

```diff
@@ -13,6 +13,11 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[3.1] - 2021-01-25
```

Comment:
Nit: 3.1.0 (or 3.0.1 since it is a bug fix?).

Reply:
edx-lint has a different versioning scheme :-/ https://pypi.org/project/edx-lint/#history

Reply:
Are you basing this on the single `1.6`, or other?  Note that we now have an OEP stating that we should use Semantic Versioning: https://open-edx-proposals.readthedocs.io/en/latest/oep-0047-bp-semantic-versioning.html, unless a repo specifically states that it wouldn't.

Reply:
> Unless there is a strong reason that I am missing to do things differently, please update and I will merge. Thanks.

I'm all for semver, but this project has so far used only x.y versions, as indicated in the edx-lint release history. I can attempt to change that but I'm not sure how the change will affect the order of pypi releases...

Reply:
I'm sure I must be missing something.  When I follow the history link you sent above, I see: 
* 3.0.1
* 3.0
* 1.6
* 1.5.2
* 1.5.0
* 1.4.1
* 1.4.0
* etc.

Only the very recent 3.0 and 1.6 switched patterns from most of the history, and these seem like exceptions, rather than the rule.  Sorry if I am still missing something obvious.

Note: Someone else snuck in, so we would now need to use 3.0.2.

Reply:
> I'm sure I must be missing something.

Ok I guess I am the one who had tunnel vision for a minute :sweat_smile: I'll bump to 3.0.2.